bin/varnishd/cache/cache_vrt.c

```diff
@@ -770,6 +770,8 @@ VRT_REAL_string(VRT_CTX, VCL_REAL num)
 {
 
 	CHECK_OBJ_NOTNULL(ctx, VRT_CTX_MAGIC);
+	if (isnan(num))
+		return (WS_Printf(ctx->ws, "NAN"));
```

Comment:
Why allocate a static string from the workspace instead of returning a static string?

Reply:
how could my efficiency fetish miss this ;)